### PR TITLE
mkcloud/qa_crowbarsetup: add debug options

### PR DESCRIPTION
**Pull request by MaximilianMeister**
* debug_mkcloud
* debug_qa_crowbarsetup
* debug_openstack

### CODE REVIEW

**Comment by aspiers** [comment]
Nice idea! +1 (although I didn't look *too* closely)

File: scripts/mkcloud [hunk 1]
```diff
@@ -190,7 +203,9 @@ pv_cur_device_no=0
 mnt=/tmp/cloudmnt/$$
 sshopts="-oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
 scp="scp $sshopts"
-: ${debug:=0}
+: ${debug_mkcloud:=0}
+: ${debug_qa_crowbarsetup:=0}
+: ${debug_openstack:=0}
```
Comment (bmwiedemann) [inline scripts/mkcloud]: I guess, you dont really need all these defaults.

**Comment by MaximilianMeister** [comment]
@bmwiedemann not sure why github does it but when you comment in the `Commits` tab it won't show the code with your comment here in the timeline. This works only when commenting in the `Files changed` tab.

**Comment by aspiers** [comment]
@MaximilianMeister It shows @bmwiedemann's comments for me.  However they will vanish from this view if the commit is rewritten and force-pushed.

**Comment by MaximilianMeister** [comment]
@aspiers strange, sure they will be marked as outdated, but should be still visible. but they are completely vanished now... just one comment is left

> I guess, you dont really need all these defaults.

**Comment by bmwiedemann** [comment]
:+1:

**Comment by MaximilianMeister** [comment]
sorry, didn't set my global git user yet, i just did a `--reset-author`

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -8,6 +8,11 @@ if [ -z "$testfunc" ] && [ -e $mkcconf ]; then
     source $mkcconf
 fi
 
+# this needs to be after mkcloud.config got sourced
```
Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: ...and pulling this down a bit

File: scripts/qa_crowbarsetup.sh [hunk 3]
```diff
@@ -8,6 +8,11 @@ if [ -z "$testfunc" ] && [ -e $mkcconf ]; then
...
+if [[ $debug_qa_crowbarsetup = 1 ]] ; then
+    set -x
+fi
```
Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: ...and pulling this down a bit

**Comment by jdsn** [comment]
:+1: nice idea

**Pull request merged**